integration/kubernetes/k8s-number-cpus.bats

```diff
@@ -17,6 +17,7 @@ setup() {
 	get_pod_config_dir
 }
 
+# Skip on aarch64 due to missing cpu hotplug related functionality.
```

Comment:
Is there a Kata tracking issue for this you could reference?

This test already has a `skip` but since `bats` is basically `bash` you could always add *another* skip below it checking for that arch and pointing to the aarch64-specific issue. Something like:

```sh
skip "test not working see: ${issue}"

arch=$(uname -m)
aarch64_issue="http://github.com/..."
[ "$arch" = aarch64 ] && skip "test disabled on arch $arch due to missing hotplug functionality: see $aarch64_issue"
```

Reply:
Hi @jodh-intel sorry for the delayed response.
Yep, I didn't create a specific tracking issue, maybe I should...it's quite a big missing feature.
I need to skip a few k8s tests on aarch64, and in order to avoid inserting multiple arch-specific `skip` command.
 ```
"$arch" = aarch64 ] && skip "test disabled on arch $arch due to missing hotplug functionality: see $aarch64_issue"
```
I create this file [configuration_aarch64.yaml](https://github.com/kata-containers/tests/blob/master/.ci/aarch64/configuration_aarch64.yaml) to record those skip tests. ;)